content/documentation/glossary/networking/index.adoc

```diff
@@ -0,0 +1,67 @@
+---
+title: "Networking"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Networking Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Networking Glossary Content
+:keywords: Kiali Documentation Glossary
+:icons: font
+
+== Destination
+
+Read the <<Destination>> definition in link:https://istio.io/help/glossary/#destination[Istio Glossary]
+
+=== Destination Rule
+
+Read the <<Destination Rule>> definition in link:https://istio.io/docs/reference/config/istio.networking.v1alpha3/#DestinationRule[Istio Destination Rule]
+
+
+== Endpoint
+
+A communication endpoint is a type of communication network node. It is an interface exposed by a communicating party or by a communication channel
+
+== Error Rate
+
+It's the percentage the errors in the traffic to a specific object in a <<Rate Interval>>
```

Comment:
--> "It's the percentage of traffic errors to a specific object, for a <<Rate Interval>>."

Also, is Rate Interval correct, or should it just be "Time Range"?

Reply:
I wrote Rate interval because is the parameter name in the UI of Kiali